From 96a2d1efdfaf5c048be587c3e9a28d526faac50f Mon Sep 17 00:00:00 2001
From: Dylan Van Assche <me@dylanvanassche.be>
Date: Fri, 12 Jul 2024 20:35:33 +0200
Subject: [PATCH] data: pine64-pinephone-*.toml: disable VoLTE by default

VoLTE has serious problems with e-calls in Germany for example.
Users can just remove the comment in the config to enable it again.

See https://gitlab.com/mobian1/issues/-/issues/254
See https://gitlab.com/postmarketOS/pmaports/-/issues/1638
See https://gitlab.com/postmarketOS/pmaports/-/issues/1624
---
 data/pine64,pinephone-1.0.toml | 4 ++--
 data/pine64,pinephone-1.1.toml | 4 ++--
 data/pine64,pinephone-1.2.toml | 4 ++--
 data/pine64,pinephone-pro.toml | 4 ++--
 4 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/data/pine64,pinephone-1.0.toml b/data/pine64,pinephone-1.0.toml
index 52df87d..d0db2f3 100644
--- a/data/pine64,pinephone-1.0.toml
+++ b/data/pine64,pinephone-1.0.toml
@@ -42,8 +42,8 @@ configure = [
     { cmd = "QDAI", expect = "1,1,0,1,0,0,1,1" },
 # RI signaling using physical RI pin
     { cmd = "QCFG", subcmd = "risignaltype", expect = "\"physical\"" },
-# Enable VoLTE support
-    { cmd = "QCFG", subcmd = "ims", expect = "1" },
+# Enable VoLTE support: remove comment to enable it
+#    { cmd = "QCFG", subcmd = "ims", expect = "1" },
 # Disable APREADY for PP 1.0 because pin is not connected
     { cmd = "QCFG", subcmd = "apready", expect = "0,0,500" },
 # URC configuration for PP 1.0 (APREADY pin not connected):
diff --git a/data/pine64,pinephone-1.1.toml b/data/pine64,pinephone-1.1.toml
index acd8751..4114b5b 100644
--- a/data/pine64,pinephone-1.1.toml
+++ b/data/pine64,pinephone-1.1.toml
@@ -42,8 +42,8 @@ configure = [
     { cmd = "QDAI", expect = "1,1,0,1,0,0,1,1" },
 # RI signaling using physical RI pin
     { cmd = "QCFG", subcmd = "risignaltype", expect = "\"physical\"" },
-# Enable VoLTE support
-    { cmd = "QCFG", subcmd = "ims", expect = "1" },
+# Enable VoLTE support: remove comment to enable it
+#    { cmd = "QCFG", subcmd = "ims", expect = "1" },
 # Disable APREADY for PP 1.1 because pin is not connected
     { cmd = "QCFG", subcmd = "apready", expect = "0,0,500" },
 # URC configuration for PP 1.1 (APREADY pin not connected):
diff --git a/data/pine64,pinephone-1.2.toml b/data/pine64,pinephone-1.2.toml
index 6f06084..e22e197 100644
--- a/data/pine64,pinephone-1.2.toml
+++ b/data/pine64,pinephone-1.2.toml
@@ -41,8 +41,8 @@ configure = [
     { cmd = "QDAI", expect = "1,1,0,1,0,0,1,1" },
 # RI signaling using physical RI pin
     { cmd = "QCFG", subcmd = "risignaltype", expect = "\"physical\"" },
-# Enable VoLTE support
-    { cmd = "QCFG", subcmd = "ims", expect = "1" },
+# Enable VoLTE support: remove comment to enable it
+#    { cmd = "QCFG", subcmd = "ims", expect = "1" },
 # Enable APREADY for PP 1.2
     { cmd = "QCFG", subcmd = "apready", expect = "1,0,500" },
 # URC configuration for PP 1.2 (APREADY pin connected):
diff --git a/data/pine64,pinephone-pro.toml b/data/pine64,pinephone-pro.toml
index 6b1348f..df5582d 100644
--- a/data/pine64,pinephone-pro.toml
+++ b/data/pine64,pinephone-pro.toml
@@ -41,8 +41,8 @@ configure = [
     { cmd = "QDAI", expect = "3,0,0,4,0,1,1,1" },
 # RI signaling using physical RI pin
     { cmd = "QCFG", subcmd = "risignaltype", expect = "\"physical\"" },
-# Enable VoLTE support
-    { cmd = "QCFG", subcmd = "ims", expect = "1" },
+# Enable VoLTE support: remove comment to enable it
+#    { cmd = "QCFG", subcmd = "ims", expect = "1" },
 # Enable APREADY for PP 1.2
     { cmd = "QCFG", subcmd = "apready", expect = "1,0,500" },
 # URC configuration for PP 1.2 (APREADY pin connected):
-- 
2.45.2

